quarkus-build-caching-extension/src/main/resources/plexus/META-INF/components.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component-set>
+    <components>
+        <component>
+            <role>com.gradle.develocity.agent.maven.api.DevelocityListener</role>
+            <role-hint>quarkus-build-cache</role-hint>
```

Comment:
I would rename it to `quarkus-build-caching-develocity-listener`.

Reply:
Addressed in a4dbcc9d468b667b26bdc6ee54881a06be24104f